src/com/oltpbenchmark/util/PlacementBlock.java

```diff
@@ -0,0 +1,47 @@
+package com.oltpbenchmark.util;
+
+public class PlacementBlock {
```

Comment:
please add a descriptive comment to this class. also any other classes you introduce in this PR. i just noticed and thought to comment on it here because its actually not clear what this does at a glance

Reply:
Done